sdk/storage/ci.yml

```diff
@@ -13,18 +26,33 @@ pr:
   branches:
     include:
     - master
+    - feature/*
+    - hotfix/*
+    - release/*
   paths:
     include:
     - sdk/storage/
 
-resources:
-  repositories:
-    - repository: azure-sdk-tools
-      type: github
-      name: Azure/azure-sdk-tools
-      endpoint: azure
-
-jobs:
-- template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+stages:
+- template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: storage
+    ArtifactName: packages
+    Artifacts:
+    - name: Azure.Storage.Blobs
+      safeName: AzureStorageBlobs
+    - name: Azure.Storage.Blobs.Batching
+      safeName: AzureStorageBlobsBatching
```

Comment:
Maybe shipping as Preview?

Reply:
Fine - we can leave it in there. Can you check the version numbers that have been specified in the *.csproj files?

Reply:
All the version numbers are correct and bumped one ahead of our last preview release.